tests/psa-client-server/psasim/tools/psa_autogen.py

```diff
@@ -111,28 +122,28 @@
         versions = versions + "};\n"
         policy = policy + "};\n"
 
-        pids.close()
-        sids.close()
-        man.close()
+        with open(MANIFEST_FILE, "wt") as output:
+            output.write("".join(manifest_content))
```

Comment:
Should that be `output.write("\n".join(manifest_content))`

Don' t we need each of those entries to be on its' own line? 

Same with SID and PID files bellow.

Reply:
> Should that be output.write("\n".join(manifest_content)) ?

It actually works because I just replaced `sid.write()` with `sid_content.append()`, but I didn't change the content of the parenthesis, so what was generated before it will be also now. However following Tom's [suggestion](https://github.com/Mbed-TLS/mbedtls/commit/8294440ad44eaf1f6485112bccb2ef419318848b#r142099566)  I can remove new lines from the parenthesis so later there will be `"".join()` as you suggested.